src/osd/OSD.h

```diff
@@ -877,6 +878,8 @@ class OSDService : public Scrub::ScrubSchedListener {
   bool prepare_to_stop();
   void got_stop_ack();
 
+  // -- PG timer --
+  common::intrusive_timer pg_timer;
```

Comment:
To be introduced in Crimson as well later on, right?

Reply:
Yeah, I actually have most of the implementation, but have had other priorities.